CMakeLists.txt

```diff
@@ -3,8 +3,12 @@ set(OpenGL_GL_PREFERENCE LEGACY)
 set(USE_QT5 FALSE CACHE BOOL "Use Qt5 instead of Qt4")
 set(USE_NEURAL_CALIBRATION FALSE CACHE BOOL "Enable neural network calibration (requires OpenCV 2, not compatible with later versions)")
 set(USE_DC1394 TRUE CACHE BOOL "Compile with DC1394 driver (firewire cameras)")
-set(USE_SPINNAKER FALSE CACHE BOOL "Compile with Spinnaker driver (FLIR USB3 cameras)")
+set(USE_SPINNAKER FALSE CACHE BOOL "Compile with Spinnaker driver (FLIR cameras)")
```

Comment:
I think the Spinnaker option should be true by default, since it will be the official version at the competition from now on.

Reply:
if it is on by default, you have to either install the SDK or disable it explicitly. Since most teams will still not use it in their labs, I would rather keep it off by default.
One option would be, to use it only, if the SDK was found, like it has been before. But I prefer the explicit way, so that the build will fail early, if it switched on, but the SDK was not found.

Reply:
Okay.